addon/bootstrap.js

```diff
@@ -3,15 +3,51 @@
 
 const OLD_ADDON_PREF_NAME = "extensions.jid1-NeEaf3sAHdKHPA@jetpack.deviceIdInfo";
 const OLD_ADDON_ID = "jid1-NeEaf3sAHdKHPA@jetpack";
+const ADDON_ID = "pageshot@mozilla.org";
+const TELEMETRY_PREF = "toolkit.telemetry.enabled";
+const USER_ENABLE_PREF = "extensions.pageshot.userdisabled";
+const SYSTEM_ENABLE_PREF = "extensions.pageshot.systemdisabled";
 
-const prefs = Components.classes["@mozilla.org/preferences-service;1"]
-  .getService(Components.interfaces.nsIPrefBranch);
 Components.utils.import("resource://gre/modules/AddonManager.jsm");
+Components.utils.import("resource://gre/modules/Services.jsm");
+const { EmbeddedExtensionManager } = Components.utils.import("resource://gre/modules/LegacyExtensionsUtils.jsm");
+
+const prefs = Services.prefs;
```

Comment:
```
/home/travis/build/mozilla-services/pageshot/addon/bootstrap.js
  15:15  error  'Services' is not defined  no-undef
  78:19  error  'Services' is not defined  no-undef
```

Not sure if we need to tweak the ESLint envs, or just manually add another `globals` to appease the ESLint gods.